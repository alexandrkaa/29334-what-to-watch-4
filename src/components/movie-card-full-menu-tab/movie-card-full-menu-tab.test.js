import React from 'react';
import renderer from 'react-test-renderer';
import MovieCardFullMenuTab from './movie-card-full-menu-tab.jsx';

const mockTab = {
  id: `TAB1`,
  name: `Tab1`,
  isActive: true,
};

it(`<MovieCardFullMenu /> should match snapshot`, () => {
  const onClickMovieCardFullMenuTab = jest.fn();
  const tree = renderer
    .create(
        <MovieCardFullMenuTab
          onClickMovieCardFullMenuTab={onClickMovieCardFullMenuTab}
          tab={mockTab}
          key={mockTab.id}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
