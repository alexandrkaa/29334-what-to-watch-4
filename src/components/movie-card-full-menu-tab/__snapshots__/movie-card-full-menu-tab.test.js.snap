// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieCardFullMenu /> should match snapshot 1`] = `
<li
  className="movie-nav__item movie-nav__item--active"
>
  <a
    className="movie-nav__link"
    href="#"
    onClick={[Function]}
  >
    Tab1
  </a>
</li>
`;
